schema.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
       required: ['createdAt']
     }
   },
-  required: ['id', 'correlationId', 'meta', 'ack', 'payload'],
+  required: ['id', 'correlationId', 'meta'],
```

Comment:
'meta' also accepts null according to the schema: 

```json
    "meta": {
      "type": ["object", "null"],
```

Should we remove 'required' for it as well?